platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/textureview/TextureViewMapRenderer.java

```diff
@@ -43,6 +43,7 @@ public TextureViewMapRenderer(@NonNull Context context,
   @Override
   protected void onSurfaceCreated(GL10 gl, EGLConfig config) {
     super.onSurfaceCreated(gl, config);
+    hasSurface = true;
```

Comment:
Shouldn't this line be in the `MapRenderer#onSurfaceCreated`? Otherwise, I believe that `GLSurfaceViewMapRenderer` never sets the value to true.